index.html

```diff
@@ -41,5 +41,159 @@ <h3 class="connect">LET'S CONNECT</h3>
+            <li><img class="dot" alt="" width="16px" height="16px" src="images/dot1.png"></li>
+            <li><p class="bed poppins">Back End Dev</p></li>
+            <li><p><img class="dot" alt="" width="16px" height="16px" src="images/dot1.png"></p></li>
+            <li><p class="year poppins">2015</p></li>
+        </ul>
+        <p class="desc2 poppins">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="flex2">
+            <li><p class="stack poppins">HTML</p></li>
+            <li><p class="stack poppins">CSS</p></li>
+            <li><p class="stack poppins">JavaScript</p></li>
+        </ul>
+        <nav>
+            <button type="button" class="see-project poppins">See project</button>
+        </nav>
+      </div>
+    </div>
+  </section>
+  <section class="about_myself">
+    <h2 class="about poppins">About Myself</h2>
+    <p class="desc poppins">Hello I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+    </p>
+    <h3 class="connect poppins">LET'S CONNECT</h3>
+    <ul>
+      <div class="container-4">
+          <li>
+            <a class="github" href="https://www.github.com" rel="noopener noreferrer" target="_blank"><img alt="github" width="18px" height="20px" src="images/github.png"></a>
+          </li>
+          <li>
+            <a class="linkedin" href="https://www.linkedin.com" rel="noopener noreferrer" target="_blank"><img class="lin" alt="linkedin" width="20px" height="20px" src="images/linkedin.png"></a>
+          </li>
+          <li>
+            <a class="angellist" href="https:wellfound.com" rel="noopener noreferrer" target="_blank"><img alt="logo-sign" width="20px" height="22px" src="images/angellist.png"></a>
+          </li>
+          <li>
+              <a class="twitter" href="https://www.twitter.com" rel="noopener noreferrer" target="_blank"><img alt="twitter" width="20px" height="20px" src="images/twitter.png"></a>
+          </li>
+          <li>
+              <a class="m" href="https://medium.com/" rel="noopener noreferrer" target="_blank"><img alt="logo-m" width="22px" height="20px" src="images/m.png"></a>
+          </li>
+      </div>
+    </ul>
+    <nav>
+      <button type="button" class="my-resume">Get my resume</button>
+    </nav>
+
+    <div class="languages">
+      <p class="lang poppins">Languages</p>
+      <button name="button-arrow1" type="button"><img class="arrow-down" alt="" width="512px" height= "512px" src="images/down-arrow.png"></button>
+    </div>
+    <ul class="lang-sub">
+      <div class="lang1">
+          <li class="li-sp">
+              <div class="row1">
+                  <img class="js-img" alt="JS Logo" width="100px" height= "100px" src="images/js.png">
+                  <p class="js poppins">JavaScript</p>
```

Comment:
Since the languages, skills and frameworks seem to be a list, I think it will be great to use the `ul` and `li` tag here to be semantically correct.